gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.5.1-bin.zip
```

Comment:
Why the downgrade to 5.5.1?

Reply:
No particular reason, you just forgot to add the wrapper and I created a new one with my version.
I will revert to 5.6.

Reply:
Does this actually work?
```
➜  flink-gradle git:(aheise-gradle-poc) ✗ ./gradlew build
Error: Could not find or load main class org.gradle.wrapper.GradleWrapperMain
```